src/components/ResumeList/type.ts

```diff
@@ -0,0 +1,7 @@
+export interface BoxProps {
```

Comment:
1. Change interface to a type - it will be more consistent with the rest of the project.
2. Maybe `ResumeListProps` could be a more meaningful name for this type.
3. Extend the type by the `BoxProps`